apps/explorer/src/lib/utils/analytics/amplitude.ts

```diff
@@ -12,15 +12,20 @@ const IS_PROD_ENV = import.meta.env.PROD;
 
 export const persistableStorage = new PersistableStorage<UserSession>();
 
+const ApiKey = {
+    production: '896b9073219c06800d9bf0aecf1b6f80',
+    development: '253fa1582d8ed913d8c5957f601df3fe',
```

Comment:
I assume this API key is public, right? 🥲

Reply:
Yes, I already told you in slack, these apis keys are public, whats private is called API Token.